content/ai/concepts/ai-models/models/ai-diffusion-models/ai-diffusion-models.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: 'Diffusion Models'
+Description:'Diffusion Models are generative models, which means they are used to generate data similar to what they were trained on. The models work by destroying training data through the addition of Gaussian noise, and then learning to recover that data.' 
+Subjects:
+  - 'Machine Learning'
+  - 'Data Science'
+Tags:
+  - 'AI'
+  - 'Image Generation Model' 
+  - 'Denoising'
+  - 'Prompts'
+  - 'Prompt Engineering'
+  - 'Text-To-Image'
+  - 'Deep Learning'
+CatalogContent:
+  - 'machine-learning'
+  - 'paths/data-science'
+---
+
+**AI Diffusion Models** Are generative models, which means they are used to generate data similar to what they were trained on. The models work by destroying training data through the addition of Gaussian noise, and then learning to recover that data.
+
+### About Diffusion Models
+- A Diffusion model is a deep learning generative model that works by adding noise to the available training data known as the forward diffusion process, it then reverses the process to recover the data. This is known as denoising or the reverse diffusion process.
+
+- The three fundamental mathematical frameworks that support the science behind diffusion models, work on the same principles of adding noise and removing it to generate new samples. 
+
+- **DDPMs** : Denoising Diffusion Probabilistic Models (generative models mainly used to remove noise from visual or audio data).
+
+- **SGMs** : Noise-Conditioned Score-Based Generative Models (can generate new samples from a given distribution. They work by learning an estimation score function that can estimate the log density of the target distribution). 
+*In english this means buiding of an estimate based on observed data of which we would like to sample*.
+
+- **SDEs** : Stochastic Differential Equations (describe changes in random processes concerning time. Widely used in physics and financial markets involving random factors that significantly impact market outcomes).
```

Comment:
```suggestion
A Diffusion model is a deep learning generative model that works by adding noise to the available training data known as the forward diffusion process, it then reverses the process to recover the data. This is known as denoising or the reverse diffusion process.

The three fundamental mathematical frameworks that support the science behind diffusion models, work on the same principles of adding noise and removing it to generate new samples. 

- **DDPMs** : Denoising Diffusion Probabilistic Models (generative models mainly used to remove noise from visual or audio data).
- **SGMs** : Noise-Conditioned Score-Based Generative Models (can generate new samples from a given distribution. They work by learning an estimation score function that can estimate the log density of the target distribution). In english this means buiding of an estimate based on observed data of which we would like to sample.
- **SDEs** : Stochastic Differential Equations (describe changes in random processes concerning time. Widely used in physics and financial markets involving random factors that significantly impact market outcomes).

```
Please follow all the edits in the suggestion. (You can just click on the commit suggestion button.)

Reply:
done